prep-html-script-tag/prep-html-script-tag-quiz-notes.md

```diff
@@ -0,0 +1,39 @@
+# prep-html-script-tag-quiz-notes
+
+## Quiz Questions
+
+Answer the following questions in the provided markdown file before turning in this exercise:
+
+- What tag is needed to load JavaScript scripts into an HTML document?
+
+- How do you use a script tag to write JavaScript directly in the HTML document?
```

Comment:
When the script is internal, it is placed directly between <script> and </script>.
<script>
</script>